src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java

```diff
@@ -98,8 +96,7 @@ public class GTRecipeTypes {
             .setSlotOverlay(false, false, GuiTextures.FURNACE_OVERLAY_1)
             .setProgressBar(GuiTextures.PROGRESS_BAR_ARROW, LEFT_TO_RIGHT)
             .setSteamProgressBar(GuiTextures.PROGRESS_BAR_ARROW_STEAM, LEFT_TO_RIGHT)
-            .setSound(GTSoundEntries.FURNACE)
-            .setXEIVisible(false);
```

Comment:
why add these to recipe viewers?

Reply:
XEI invisible = nothing gets registered to XEI, not even workstations
I have removed the furnace recipe category from the `RECIPE_CATEGORIES` registry, so the electric furnace recipes will not be registered to recipe viewers. Also have added a special case in each of the `GTXEICategory` classes.

Reply:
This lets us remove the hardcoded furnace workstation registration.